docs/examples/PanelCollapsible.js

```diff
@@ -12,7 +12,11 @@ class Example extends React.Component {
         <Button onClick={ ()=> this.setState({ open: !this.state.open })}>
           click
         </Button>
-        <Panel collapsible expanded={this.state.open}>
+        <Panel
+          collapsible
+          id="collapsible-panel-example"
+          expanded={this.state.open}
+        >
           Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid.
```

Comment:
took a quick stab at it. You don't end up saving much unless you also want to get rid of `<Panel collapsible>` as well, b/c you still need to loop through the child to find the non header/footer elements to wrap in PanelCollapse.

This: `<Panel.Body collapse>` is an alternative but it doesn't work with mixed body elements like a table and panel-body. I'm not sure which use-case is more common but the worst case looks like this now:

```
<Panel>
  <Panel.Collapse>
    <Panel.Body>fooobar</Panel.Body>
  </Panel.Collapse> 
<Panel>
```

which sucks